hooks/pre-command

```diff
@@ -61,8 +61,10 @@ if [[ ! -d "${DOWNLOAD_DESTINATION}" ]]; then
 fi
 
 URL="https://github.com/vmware-tanzu/carvel-ytt/releases/download/v${YTT_VERSION}/ytt-${OS}-${ARCH}"
+echo "Downloading $URL..."
```

Comment:
You need a `.exe` at the end of this for windows.  :)

Reply:
Fixed in b1da3dc!